src/main/java/uk/gov/justice/hmpps/probationteams/model/LocalDeliveryUnit2.java

```diff
@@ -41,6 +44,16 @@
     @Column
     private String functionalMailbox;
 
+    /**
+     * Map of probation team code to ProbationTeam
+     */
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(
+            name = "PROBATION_TEAM",
+            joinColumns = @JoinColumn(name = "LOCAL_DELIVERY_UNIT_ID"))
+    @MapKeyColumn(name = "TEAM_CODE")
+    private Map<String, ProbationTeam> probationTeams = Map.of();
```

Comment:
meh.. not sure if it matters but we don't often default empty collections - if we do sometimes we'll add the lombok annotation to use this as a default for the builder?